_docs_integrate/13-connector-error-codes.md

```diff
@@ -0,0 +1,79 @@
+| {% include custom-anchor name="error.transport.files.invalidTruncatedReference" %}                                    | The given truncated reference is invalid.                                                                                                                                 |
+| {% include custom-anchor name="error.transport.files.maxFileSizeExceeded" %}                                          | The given file content size exceeds the max file size the backbone accepts.                                                                                               |
+| {% include custom-anchor name="error.transport.files.plaintextHashMismatch" %}                                        | The actual hash of the plaintext does not match the given plaintextHash. Something went wrong while encrypting/decrypting the file.                                       |
+| {% include custom-anchor name="error.transport.general.baseUrlNotSet" %}                                              | The baseUrl was not set.                                                                                                                                                  |
+| {% include custom-anchor name="error.transport.general.platformClientIdNotSet" %}                                     | The platform client id was not set.                                                                                                                                       |
+| {% include custom-anchor name="error.transport.general.platformClientInvalid" %}                                      | The combination of platform clientId and clientSecret is invalid.                                                                                                         |
+| {% include custom-anchor name="error.transport.general.platformClientSecretNotSet" %}                                 | The platform client secret was not set.                                                                                                                                   |
+| {% include custom-anchor name="error.transport.identity.realmLength" %}                                               | The given realms is not 3 characters long.                                                                                                                                |
+| {% include custom-anchor name="error.transport.incompatibleBackbone" %}                                               | The backbone sent an invalid payload.                                                                                                                                     |
+| {% include custom-anchor name="error.transport.messages.noMatchingRelationship" %}                                    | No Relationship was found for a specific sender.                                                                                                                          |
```

Comment:
Der Error Code wird in zwei sehr unterschiedlichen Kontexten genutzt: `getMessagesByAddress` und `decryptMessage`. Auch wenn der Error Code selbst zwar in beiden Kontexten passt: die Error Message tut es nicht.

Bei `decryptMessage` ist es einigermaßen richtig (wobei hier das "specific" in der Fehlermeldung irgendwie komisch ist; ich hätte geschrieben "No Relationship was found to the sender of the Message.").

Bei `getMessagesByAddress` aber passt die Message gar nicht. Also entweder die Error Message als Parameter übergeben, oder einen anderen/spezifischeren Error Code für beide verwenden.

Reply:
msg ist überarbeitet